students/heewon-moon/westagram/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class User(models.Model):
+    email       = models.EmailField(max_length=200, unique=True)
+    password    = models.CharField(max_length=200)
+    nickname    = models.CharField(max_length=100, unique=True)
+    phone       = models.CharField(max_length=100, unique=True)
+    created_at  = models.DateTimeField(auto_now_add=True)
+    updated_at  = models.DateTimeField(auto_now=True)
```

Comment:
```suggestion
    email      = models.EmailField(max_length=200, unique=True)
    password   = models.CharField(max_length=200)
    nickname   = models.CharField(max_length=100, unique=True)
    phone      = models.CharField(max_length=100, unique=True)
    created_at = models.DateTimeField(auto_now_add=True)
    updated_at = models.DateTimeField(auto_now=True)
```
가장 긴 변수명 기준으로 한 칸 정도만 띄우면 될 것 같습니다.